tests/test_trajectory.py

```diff
@@ -719,16 +719,19 @@ def test_smooth(get_fn):
 
     eq(output, test)
 
-@pytest.mark.skip(reason="Broken, maybe only on Python 3.11")
 def test_image_molecules(get_fn):
     # Load trajectory with periodic box
     t = md.load(get_fn('alanine-dipeptide-explicit.dcd'), top=get_fn('alanine-dipeptide-explicit.pdb'))
     # Image to new trajectory
     t_new = t.image_molecules(inplace=False)
+    # Test for issue #1769
```

Comment:
Quick small comment here upon reread - could you edit this comment also to specify what this test is testing for (keeping 1769 is fine). For long term it'd be good to ensure that the code documents _why_ we're testing this and what the expectation is.

Reply:
I have edited this comment too:
```
# Test that t_new and t are not the same object (issue #1769)
```